upgrade_pythoncapi.py

```diff
@@ -563,11 +575,15 @@ def _parse_options(self, args):
         parser.add_argument(
             '-C', '--no-compat', action="store_true",
             help=f"Don't add: {INCLUDE_PYTHONCAPI_COMPAT}")
+        parser.add_argument(
+            '-d', '--download',
+            help=f'Download latest pythoncapi_compat.h file to designate path',
```

Comment:
maybe use metavar='PATH' to use:

```suggestion
            help=f'Download latest pythoncapi_compat.h file to designate PATH',
```

See how --help generates the documentation.